.travis.yml

```diff
@@ -0,0 +1,15 @@
+after_success:
+  - $(npm bin)/update-branch --commands "npm run build"
+                             --commit-message "Hey GitHub, this content is for you! [skip ci]"
+                             --directory "out"
+                             --distribution-branch "gh-pages"
+                             --source-branch "master"
+
+env:
+  global:
+    - secure: "<secure_key>"
+
+language: node_js
+
+node_js:
+  - '5'
```

Comment:
Can we not mix single/double quotes? This is the only single-quoted value in this file.